examples/mobile/UIComponents/webclip/link-tab/images/tw_list_icon_connections.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Where is svg used?

Reply:
> Where is svg used?

Yes :) . File ` examples/mobile/UIComponents/webclip/link-tab/webclip.html` line 57.